### PR TITLE
Add Contact App, update Base HTMl file to inlude 'Contact' page in Navbar, add content to Contact page

### PR DESCRIPTION
**Pull request by ProfessorJames**
1.  Add Contact app. 
- Add to installed apps in setting.py. 
- Create folder structure for Contact app, add in static files.  
- Base.html: 
      - add link for Contact page to Navbar, update fontawesome icon to envelope. 
- Migrate Contact models.

2.  Add content to Contact page.
 
Note - 2 reviewers required for this pull request please

### CODE REVIEW

**Comment by e-kai00** [comment]
Looks good to me. I only did not understand why to use model. We probably don't need (?) But let's ask others, what they think.

**Comment by ProfessorJames** [comment]
Probably no need for a model. Was following what was done for About app. If ye wnat ot removed just let me know.

File: contact/views.py
Comment (KimBergstroem) [inline contact/views.py:1]: Double import of render, should be remove one of them atleast.

File: contact/tests.py
Comment (KimBergstroem) [inline contact/tests.py:1]: Remove the tests.py file completely. I will do so also for others, but you can do it for this app.

File: contact/models.py
Comment (KimBergstroem) [inline contact/models.py:1]: the class name is confusing for me, is this about me section or is it going to be a contact form?

Comment (ProfessorJames) [reply contact/models.py:1]: its a contact page with a form, and vaarious contact information
Screenshot 2024-01-20 134048

Comment (e-kai00) [reply contact/models.py:1]: looks awesome )) 
@KimBergstroem I did not get it - should we approve it, or smth should be edited before we approval?

Comment (ProfessorJames) [reply contact/models.py:1]: I can make the above changes outlined by Kim. Before I do though, can ye confirm if the class name needs to be changed/updated? and if so what would be an appropriate name now that ye have seen the content of the Contact page and have more context.

**Review by e-kai00 — APPROVED** [review]
GLTM

**Pull request merged**